references/classification/train_quantization.py

```diff
@@ -233,6 +268,12 @@ def get_args_parser(add_help=True):
         help="Post training quantize the model",
         action="store_true",
     )
+    parser.add_argument(
+        "--quantization-workflow-type",
```

Comment:
Why not have a `--use-fx-quantization` boolean flag instead? Do we expect more types of quantization on the future?

Reply:
Yes, actually in the original changes we had a simple `--fx` flag. However, in the future we may add support for `define_by_run_quantization` in addition to the two existing workflow types.